traitsui/testing/tester/qt4/implementation/ui_base.py

```diff
@@ -0,0 +1,36 @@
+#  Copyright (c) 2005-2020, Enthought, Inc.
+#  All rights reserved.
+#
+#  This software is provided without warranty under the terms of the BSD
+#  license included in LICENSE.txt and may be redistributed only
+#  under the conditions described in the aforementioned license.  The license
+#  is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+#  Thanks for using Enthought open source!
+#
+from traitsui.qt4.ui_base import ButtonEditor
+from traitsui.testing.tester import command, query
+from traitsui.testing.tester.qt4 import helpers
+
+
+def register(registry):
+    """ Register solvers/handlers specific to qt ui_base Editors
+    for the given registry.
+
+    If there are any conflicts, an error will occur.
+
+    Parameters
+    ----------
+    registry : TargetRegistry
+    """
+    handlers = [
+        (command.MouseClick, (lambda wrapper, _:  helpers.mouse_click_qwidget(
+                              wrapper.target.control, wrapper.delay))),
+        (query.DisplayedText, lambda wrapper, _: wrapper.target.control.text())
```

Comment:
This handler for `DisplayedText` not used in the test, maybe it is needed in the future but I think it is better to defer it until we actually have a use case to use it. It is better to avoid speculative generality, this is especially true when it comes to something that will become a published API.

Reply:
Or if we change the following test to use DisplayedText, then this represents an existing use case and we can keep this handler here:
https://github.com/enthought/traitsui/blob/24bcf02fe2216237fd207b4f679629416d073754/traitsui/tests/test_ui.py#L191